webapp/client/src/components/BackLink.js

```diff
@@ -10,6 +10,7 @@ const Anchor = styled.a`
   text-transform: uppercase;
   letter-spacing: var(--tracking-extra-wide);
   text-decoration: none;
+  padding-top: 0.3rem;
```

Comment:
Do we still have to set the padding for the LinkElement (l. 54)?
```css
const LinkElement = styled.span`
  color: var(--text-color);
  border-bottom: 1px solid transparent;
  padding: 3px 3px 0;
`;
```

Reply:
Yes, it is because of the different icon sizes. I follow the Figma Design and asking for feedback from UX.